### PR TITLE
[OSDOCS-3598]: Nutanix IPI CCO setup

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.11+

Issue:
[OSDOCS-3598](https://issues.redhat.com/browse/OSDOCS-3598)

Link to docs preview: (VPN required)
- [Configuring the Cloud Credential Operator utility](http://file.rdu.redhat.com/jrouth/OSDOCS-3598-Nutanix-CCO-IPI/installing/installing_nutanix/preparing-to-install-on-nutanix.html#cco-ccoctl-configuring_preparing-to-install-on-nutanix)
- [Configuring IAM for Nutanix](http://file.rdu.redhat.com/jrouth/OSDOCS-3598-Nutanix-CCO-IPI/installing/installing_nutanix/installing-nutanix-installer-provisioned.html#manually-create-iam-nutanix_installing-nutanix-installer-provisioned)

Additional information:
This PR is intended to facilitate review of the Nutanix CCO config process.

### CODE REVIEW

File: modules/manually-configure-iam-nutanix.adoc [hunk 1]
```diff
@@ -8,54 +8,55 @@
...
+      password: <password_for_prism_element>
 ----
-<1> Add this line to set the `credentialsMode` parameter to `Manual`.
+<1> Specifies the authentication type. Only basic authentication is supported.
+<2> Specifies the Prism Central credentials.
+<3> Optional: Specify the Prism Element credentials.
++
+[NOTE]
+====
+The default location for the YAML file is in `<home_directory>/.nutanix/credentials`. To specify a different directory, use the `--credentials-source-filepath` flag. This procedure uses `<path_to_credentials_file>` to refer to this directory.
```
Comment (akhil-rane) [inline modules/manually-configure-iam-nutanix.adoc]: It is not super clear here on where to use `--credentials-source-filepath` flag. We should mention somewhere that it needs to be used with `cccotl`

Comment (akhil-rane) [reply modules/manually-configure-iam-nutanix.adoc]: May be we should put some reference to step 3?

Comment (jeana-redhat) [reply modules/manually-configure-iam-nutanix.adoc]: Ah, good idea - I can just add it as an annotation where it is used. Should probably do that for other ccoctl procedures too but will start here :)

File: modules/manually-configure-iam-nutanix.adoc [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+      password: <password_for_prism_element>
+----
+<1> Specify the authentication type. Only basic authentication is supported.
+<2> Specify the Prism Central credentials.
+<3> Optional: Specify the Prism Element credentials.
++
+[NOTE]
+====
+The default location for the YAML file is in `<home_directory>/.nutanix/credentials`. To specify a different directory, use the `--credentials-source-filepath` flag. This procedure uses `<path_to_credentials_file>` to refer to this directory.
+#The procedure below lacks any instance of `--credentials-source-filepath`. Where/when does a user need to specify this if they are not using the default directory?#
```
Comment (akhil-rane) [inline modules/manually-configure-iam-nutanix.adoc]: by default we pick credentials from <home_directory>/.nutanix/credentials. If user has credentials stored at some other place they can use `--credentials-source-filepath` flag

Comment (jeana-redhat) [reply modules/manually-configure-iam-nutanix.adoc]: Ok I think adding this info to step 3 makes sense. Updated and will refresh PR shortly.

File: modules/manually-configure-iam-nutanix.adoc [hunk 3]
```diff
@@ -78,21 +68,59 @@ $ ccoctl nutanix create-shared-secrets \
...
+[source,terminal]
+----
+$ ls ./<installation_directory>/manifests
+----
++
+.Example output
++
+[source,terminal]
+----
+#Is there a sample output here?#
```
Comment (jeana-redhat) [inline modules/manually-configure-iam-nutanix.adoc:125]: Note: would like an output here but it does not need to hold this PR up. Can omit if needed, but hopefully will be available before docs freeze.

**Comment by jeana-redhat** [comment]
Closed in favor of https://github.com/openshift/openshift-docs/pull/44537, which includes these docs in the full install flow.

**Configuring the Cloud Credential Operator utility**

- File: https://github.com/openshift/openshift-docs/pull/44537/files#diff-6f0f1b2e8660fb0a6c03a6ceee1cc76715790953c8a035a09fdc0009b6dd8271
- Preview: http://file.rdu.redhat.com/mpytlak/osdocs-3435/installing/installing_nutanix/preparing-to-install-on-nutanix.html#cco-ccoctl-configuring_preparing-to-install-on-nutanix

**Configuring IAM for Nutanix**

- File: https://github.com/openshift/openshift-docs/pull/44537/files#diff-d9134251a7dfb97bab3fd7ac50f8370be5911b6185d43e5cf3ee5362bf419a94
- Preview: http://file.rdu.redhat.com/mpytlak/osdocs-3435/installing/installing_nutanix/installing-nutanix-installer-provisioned.html#manually-create-iam-nutanix_installing-nutanix-installer-provisioned

File: modules/manually-configure-iam-nutanix.adoc (hunk 3 above)
Comment (akhil-rane) [reply modules/manually-configure-iam-nutanix.adoc:125]: here is sample output after copy command

```
[akhilrane@localhost nutanix]$ cp manifests_ccoctl/manifests/*credentials.yaml ./manifests-installer/manifests
[akhilrane@localhost nutanix]$ ls manifests-installer/manifests/
cluster-config.yaml                   cluster-network-02-config.yml    kube-system-configmap-root-ca.yaml
cluster-dns-02-config.yml             cluster-proxy-01-config.yaml     machine-config-server-tls-secret.yaml
cluster-infrastructure-02-config.yml  cluster-scheduler-02-config.yml  openshift-config-secret-pull-secret.yaml
cluster-ingress-02-config.yml         cvo-overrides.yaml               openshift-machine-api-nutanix-credentials-credentials.yaml
cluster-network-01-crd.yml            kube-cloud-config.yaml
[akhilrane@localhost nutanix]$ 

```

**Pull request closed without merging**